documentation/docs/libraries/java/how_to/_run_how_tos.md

```diff
@@ -9,20 +9,17 @@ git clone git@github.com:iotaledger/wallet.rs.git
 Then, move into the Java code by running the following command:
 
 ```bash
-cd wallet.rs/bindings/java/iota-wallet-java
+cd wallet.rs/bindings/java/examples
 ```
 
 ## Run Code Examples
 
-The IOTA Wallet Java library has numerous [examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/java/iota-wallet-java/examples/src)
-you can run to get acquainted with the library. After you have followed the instructions to
-[install the library](./../getting_started/java#install-the-library), you can run any example with the following
+The IOTA Wallet Java library has numerous [examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/java/examples/src)
+you can run to get acquainted with the library. You can run any example with the following
 command from the `examples` directory:
 
 ```bash
-./gradlew run -Pmain=CreateAccount
+.././gradlew run -Pmain=CreateAccount
```

Comment:
././ ?

Reply:
we did`cd in examples/` before but `gradlew` is is only in the outer folder
therefore we need to run it via `.././gradlew`

Reply:
Oh sorry I actually misread it, only saw `././` which was weird